src/sonic-host-services/scripts/hostcfgd

```diff
@@ -633,51 +633,15 @@ class AaaCfg(object):
             return
         chage_flag = AGE_DICT[age_type]['CHAGE_FLAG']
         for normal_account in normal_accounts:
-
-            new_curr_expiration = self.get_user_expire_time(normal_account, curr_expiration)
-
             try:
-                chage_p_m = subprocess.Popen(('chage', chage_flag + str(new_curr_expiration), normal_account), stdout=subprocess.PIPE)
+                chage_p_m = subprocess.Popen(('chage', chage_flag + str(curr_expiration), normal_account), stdout=subprocess.PIPE)
```

Comment:
Why was this modified? Does it mean expiration time would be calculated from the time set and not from previous password change?

Reply:
It will be calculated from the last password changed, When I finished the test I figured out that the library "chage" is doing it automatically from the last password changed, and I don't really need to do the calculation.